tests/puppeteer/campaigns/sanity/ordersBO/01_filterOrder.js

```diff
@@ -0,0 +1,65 @@
+// Using chai
+const {expect} = require('chai');
+// importing pages
+const LoginPage = require('../../../pages/BO/login');
+const DashboardPage = require('../../../pages/BO/dashboard');
+const BOBasePage = require('../../../pages/BO/BObasePage');
+const OrderPage = require('../../../pages/BO/order');
+
+let page;
+let loginPage;
+let dashboardPage;
+let boBasePage;
+let orderPage;
+// creating pages objects in a function
+const init = async () => {
+  page = await global.browser.newPage();
+  loginPage = await (new LoginPage(page));
+  dashboardPage = await (new DashboardPage(page));
+  boBasePage = await (new BOBasePage(page));
+  orderPage = await (new OrderPage(page));
+};
+
+/*
+  Connect to the BO
+  Filter the Orders table
+  Logout from the BO
+ */
+global.scenario('Filter the Orders table by ID, REFERENCE, STATUS', async () => {
+  test('should login into BO', async () => {
```

Comment:
We will soon have a problem with this test, since it will be reused everywhere. We need a way to have reusable scenarios.

Reply:
Yes, I think that the solution is to create a file named commonScenario contains all common tests used in all campaigns

Reply:
Maybe regroup it in common_scenarios, but it most be done for all tests, so not in this PR I think